launcher.py

```diff
@@ -84,53 +84,47 @@ def path_text():
     print("")
 
-    # MACHINE CHOICE
-machine_id_input = input("Select a machine (number), create a new one with 'c', or enter 0 to quit: ")
-
-if machine_id_input.lower() == 'c':
-    # Logic for creating a new machine
-    new_machine_name = input("Enter a name for the new machine: ")
-    new_machine_path = os.path.join(machine_path, new_machine_name)
-
-    # Check if the folder already exists
-    if os.path.exists(new_machine_path):
-        print(f"Machine '{new_machine_name}' already exists. Please choose a different name.")
-    else:
-        # Create a new folder for the machine
-        os.makedirs(new_machine_path)
-        print(f"New machine '{new_machine_name}' created in '{machine_path}'!")
-
-        # Ask if the user wants to configure the new machine
-        configure_new_machine = input("Do you want to configure this new machine? (y/n): ").lower()
-
-        if configure_new_machine == 'y':
-            # Configure the new machine
-            print("Starting 86Box for configuration...")
-            os.system(app_path + f" -S \"{machine_path}/{new_machine_name}/86box.cfg\"")
-
-        # Reload the script to detect the new machine
-        print("Reloading the script...")
-        script_path = os.path.abspath(__file__)
-        os.execv(sys.executable, [sys.executable, script_path])
-else:
     try:
-        machine_id = int(machine_id_input)
-    except ValueError:
-        print("Invalid input. Please enter a valid number or 'c' to create a new machine.")
+        # MACHINE CHOICE
+        machine_id_input = input("Select a machine (number), create a new one with 'c', or enter 0 to quit: ")
+
+        if machine_id_input.lower() == 'c':
+            # Logic for creating a new machine
+            new_machine_name = input("Enter a name for the new machine: ")
+            new_machine_path = os.path.join(machine_path, new_machine_name)
+
+            # Check if the folder already exists
+            if os.path.exists(new_machine_path):
+                print(f"Machine '{new_machine_name}' already exists. Please choose a different name.")
+            else:
+                # Create a new folder for the machine
+                os.makedirs(new_machine_path)
+                print(f"New machine '{new_machine_name}' created in '{machine_path}'!")
+
+                # Ask if the user wants to configure the new machine
+                configure_new_machine = input("Do you want to configure this new machine? (y/n): ").lower()
+
+                if configure_new_machine == 'y':
+                    # Configure the new machine
+                    print("Starting 86Box for configuration...")
+                    os.system(app_path + f" -S \"{machine_path}/{new_machine_name}/86box.cfg\"")
+
+                # Reload the script to detect the new machine
+                print("Reloading the script...")
+                script_path = os.path.abspath(__file__)
+                os.execv(sys.executable, [sys.executable, script_path])
```

Comment:
What OS did you test this on? I'm getting path-related issues on Windows.

Reply:
oh i tried on win 11

Reply:
i dont have mac rn

Reply:
idk

Reply:
i figured id make a better ver i guess its better to update with the 4.1

Reply:
i guess config doesn't work in 4.1

Reply:
its basically useless rn without the config and the only crash i have is if i enter invalid name in create but

Reply:
oh i guess it does work it justs takes 5 min

Reply:
https://github.com/DDXofficial/sl86/assets/154773078/6fc7e4cd-05c2-4e87-b179-b9e951dbee96